Tests/Common/IntegrationTestFixtureBase.Asserts.cs

```diff
@@ -65,14 +65,14 @@ public async Task AssertIoTHubDeviceMessageExistsAsync(string deviceID, string t
 
             if (assertionLevel == LogValidationAssertLevel.Error)
             {
-                var logs = string.Join("\n\t", searchResult.Logs.TakeLast(5));
+                var logs = string.Join("\n\t", searchResult.Logs.Select(x => x.ToString()).TakeLast(5));
```

Comment:
is this related?

Reply:
No, reverted now 02caca3b8ec2d9e9effca4959a2208f7b83fe480 - was trying to get a more detailed log message but it's not needed anymore.